tests/fixtures/config/minimal_contenttypes.yaml

```diff
@@ -0,0 +1,11 @@
+# Minimal Contenttypes
+
+foo:
+  name: Bars
```

Comment:
name is not needed for minimal Content Type ;)

Reply:
Yes it is. Either the name of the slug is required.

Reply:
but there is a slug in line 3

Reply:
Added another test for it.